### ISSUE TITLE
reformat pages to easier reading

### ISSUE THREAD

**Issue by 3d73a4a412** [JonasCz/save-for-offline#39]
Fantastic app. I use it constantly. The addition of a "Reader View" ([like in Firefox](https://support.mozilla.org/t5/Basic-Browsing/Firefox-Reader-View-for-clutter-free-web-pages/ta-p/38466), or the Firefox "[tranquility](http://tranquility.ushnisha.com/)" or "[reader](http://barisderin.com/?p=372)" addons [both GPL if that helps?]) would be *very* useful. Some pages are quite difficult to read and many have unuseful cruft that could be removed by a "reader view", might be useful as well for saving, that is just saving the modified view instead of the extra formatting etc. If this would be possible and not too difficult it would be really really really appreciated!

**Comment by JonasCz** [comment]
Yes, should be possible - not in the near future though, since I'm quite busy with other things.

The app includes the option to open saved pages in other apps though, you could open the page in e.g Firefox for Android with whatever extensions / options you want.

**Comment by 3d73a4a412** [comment]
Thx for the response, and fair enough. Just checking. I have a older phone (ie can't run too many aps) and mainly use SFO with redreader then opening it up with FF might be a bit much for my workflow. Perhaps I should try bugging the RedReader dev for an alt solution? ;)  ...really wish I could code. Anyway, thx again for a gr8 app, do you have a btc donation wallet?

**Comment by JonasCz** [comment]
I'm not sure RedReader dev will have any other ideas - but I'm sure there's other reader apps which are more lightweight than FF, and which may work well enough. 

Maybe I'll have time to implement something in the future though - should not be _too hard_.
